src/wp-includes/sitemaps/providers/class-wp-sitemaps-taxonomies.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Sitemaps: WP_Sitemaps_Taxonomies class
+ *
+ * Builds the sitemaps for the 'taxonomy' object type.
+ *
+ * @package WordPress
+ * @subpackage Sitemaps
+ * @since 5.5.0
+ */
+
+/**
+ * Taxonomies XML sitemap provider.
+ *
+ * @since 5.5.0
+ */
+class WP_Sitemaps_Taxonomies extends WP_Sitemaps_Provider {
+	/**
+	 * WP_Sitemaps_Taxonomies constructor.
+	 *
+	 * @since 5.5.0
+	 */
+	public function __construct() {
+		$this->name        = 'taxonomies';
+		$this->object_type = 'term';
+	}
+
+	/**
+	 * Returns all public, registered taxonomies.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array Map of registered taxonomy objects keyed by their name.
+	 */
+	public function get_object_subtypes() {
+		$taxonomies = get_taxonomies( array( 'public' => true ), 'objects' );
```

Comment:
`public` implies `rewrite`, but aren't they mutually exclusive?

Reply:
I don't see how `rewrite` is relevant here.